pyvrp/cpp/TimeWindowSegment.h

```diff
@@ -104,18 +107,22 @@ TimeWindowSegment TimeWindowSegment::merge(
 #ifdef PYVRP_NO_TIME_WINDOWS
     return {};
 #else
-    auto const travelDuration = durationMatrix(idxLast_, other.idxFirst_);
-    auto const diff = duration_ - timeWarp_ + travelDuration;
-    auto const diffWait
-        = std::max<Duration>(other.twEarly_ - diff - twLate_, 0);
-    auto const diffTw = std::max<Duration>(twEarly_ + diff - other.twLate_, 0);
+    // Arc travel from our last to other's first, and the time at which we
+    // arrive at the other's first client.
+    auto const arcDuration = durationMatrix(idxLast_, other.idxFirst_);
```

Comment:
We don't use the concept of `arc` anywhere in PyVRP but we use `travelDuration` to refer to the duration of travelling between clients... so I think this should be `travelDuration`, why did you change it back?

Reply:
We use arc here and there in the documentation, so it's not completely alien. We use edge more often, so I've renamed it to that. We use `travelDuration` only as an aggregate, not for single edges. In this context it could just as well mean "travel duration of all clients in the TWS". I find that a bit confusing.